Cpix/HlsPlaylistType.cs

```diff
@@ -3,6 +3,6 @@
 	public class HlsPlaylistType
```

Comment:
Any particular reason this is not a sealed class? Feels like it should be.

Reply:
No reason. Will seal.